sdk/runanywhere-kotlin/modules/runanywhere-core-onnx/src/jvmAndroidMain/kotlin/com/runanywhere/sdk/core/onnx/RunAnywhereBridge.kt

```
@@ -0,0 +1,299 @@
+    @Synchronized
+    fun loadLibrary() {
+        if (isLibraryLoaded) return
+
+        try {
+            // Try to load the JNI bridge library
+            System.loadLibrary("runanywhere_jni")
+
+            // Also load dependencies if needed
+            try {
+                System.loadLibrary("runanywhere_bridge")
+            } catch (e: UnsatisfiedLinkError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)